benchmarking/benchmark.py

```diff
@@ -0,0 +1,346 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_array
+
+import rdkit.Chem.rdFingerprintGenerator as fpgens
+from rdkit.Chem.rdMolDescriptors import GetMACCSKeysFingerprint
+from rdkit.Chem.rdReducedGraphs import GetErGFingerprint
+
+from time import time
+
+from featurizers.fingerprints import (
+    MorganFingerprint,
+    AtomPairFingerprint,
+    TopologicalTorsionFingerprint,
+    MACCSKeysFingerprint,
+    ERGFingerprint,
+)
+
+dataset_name = "ogbg-molhiv"
+N_SPLITS = 5
+N_REPEATS = 5
+N_CORES = [1, 2, 4, -1]
```

Comment:
Change N_CORES to run from 1 to NUM_CORES, where num cores is the number of physical cores on the machine. You can use Joblib for finding the exact number.

Reply:
Done